styles/roblox/catppuccin.user.css

```
@@ -0,0 +1,200 @@
+/* ==UserStyle==
+@name Roblox Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/roblox
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/roblox
+@version 1.0.1
```

Lets start from a whole number
```suggestion
@version 1.0.0
```